queries.sql

```diff
@@ -0,0 +1,11 @@
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
```

Comment:
**OPTIONAL** - In this query you are to list the name of all animals born between 2016 and 2019. .ie between `2016/01/01 and 2019/31/12`. Please fix this 🙏